meson.build

```diff
@@ -146,4 +146,19 @@ top_earlgrey = declare_dependency(
   )
 )
 
+# ROM_EXT manifest generator.
+rom_exts_source_dir = meson.source_root() / 'sw/device/rom_exts'
+rom_exts_manifest_gen = generator(
```

Comment:
This should use `custom_target`, so that `rom_ext_manifest.h` is not regenerated for every built target that depends on it. This is a shortcoming of `generator` which we're avoiding for non-register headers now.

Reply:
> This should use custom_target, so that rom_ext_manifest.h is not regenerated for every built target

The reasoning behind it is explained in the commit message for the change. However, as we have decided that this file should be checked it - then `custom_target` is more appropriate.

I will fix it up in the next push.

Reply:
I have made it a custom target that is built independently of other targets.

However there is a minor issue, it runs the script every time you run the script, if anyone could point out where I am going wrong there that would be helpful.